--- common-links.html.tmpl.orig	2013-04-18 10:34:18.361658028 -0700
+++ common-links.html.tmpl	2013-04-18 10:46:03.243917945 -0700
@@ -69,11 +69,6 @@
 
     <li>
       <span class="separator">| </span>
-      [% IF user.authorizer.can_logout %]
-        <a href="index.cgi?logout=1">Log&nbsp;out</a>
-      [% ELSE %]
-        Logged&nbsp;in&nbsp;as
-      [% END %]
       [% IF sudoer %]
         [%+ sudoer.login FILTER html %] (<b>impersonating
         [%+ user.login FILTER html %]</b>
