--- header.html.tmpl.orig	2021-06-09 21:34:30.951156821 +0100
+++ header.html.tmpl	2021-06-09 21:35:56.261837211 +0100
@@ -210,6 +210,7 @@
     <link rel="search" type="application/opensearchdescription+xml"
                        title="[% terms.Bugzilla %]" href="./search_plugin.cgi">
     <link rel="shortcut icon" href="[% favicon_url FILTER html %]">
+    <link rel="icon" type="image/png" href="/images/VASSAL.png"/>
     [% Hook.process("additional_header") %]
   </head>
 
@@ -219,6 +220,9 @@
                  [%+ class FILTER css_class_quote %]
                [% END %] yui-skin-sam">
 
+  [% INCLUDE "global/vassal-header.html.tmpl" %]
+  <div id="wrapper">
+
   <div id="header">
     [% INCLUDE global/banner.html.tmpl %]
 
