--- header.html.tmpl.orig	2015-06-19 19:59:06.481689939 +0200
+++ header.html.tmpl	2015-06-19 19:59:16.469247537 +0200
@@ -212,6 +212,7 @@
     <link rel="search" type="application/opensearchdescription+xml"
                        title="[% terms.Bugzilla %]" href="./search_plugin.cgi">
     <link rel="shortcut icon" href="images/favicon.ico" >
+    <link rel="icon" type="image/png" href="/images/VASSAL.png"/>
     [% Hook.process("additional_header") %]
   </head>
 
@@ -229,6 +230,9 @@
   # 'bannerhtml'
   #%]
 
+[% INCLUDE "global/vassal-header.html.tmpl" %]
+<div id="wrapper">
+
 <div id="header">
 
 [% INCLUDE global/banner.html.tmpl %]
