--- header.html.tmpl.orig	2012-03-15 18:26:26.415439373 -0700
+++ header.html.tmpl	2012-03-15 18:26:33.286434656 -0700
@@ -235,6 +235,7 @@
     <link rel="search" type="application/opensearchdescription+xml"
                        title="[% terms.Bugzilla %]" href="./search_plugin.cgi">
     <link rel="shortcut icon" href="images/favicon.ico" >
+    <link rel="icon" type="image/png" href="/images/VASSAL.png"/>
     [% Hook.process("additional_header") %]
   </head>
 
@@ -252,6 +253,9 @@
   # 'bannerhtml'
   #%]
 
+[% INCLUDE "global/vassal-header.html.tmpl" %]
+<div id="wrapper">
+
 <div id="header">
 
 [% INCLUDE global/banner.html.tmpl %]
