--- header.html.tmpl.orig	2013-04-18 10:45:50.120944217 -0700
+++ header.html.tmpl	2013-04-18 10:46:18.074888685 -0700
@@ -241,6 +241,7 @@
     <link rel="search" type="application/opensearchdescription+xml"
                        title="[% terms.Bugzilla %]" href="./search_plugin.cgi">
     <link rel="shortcut icon" href="images/favicon.ico" >
+    <link rel="icon" type="image/png" href="/images/VASSAL.png"/>
     [% Hook.process("additional_header") %]
   </head>
 
@@ -258,6 +259,9 @@
   # 'bannerhtml'
   #%]
 
+[% INCLUDE "global/vassal-header.html.tmpl" %]
+<div id="wrapper">
+
 <div id="header">
 
 [% INCLUDE global/banner.html.tmpl %]
