--- functions.php.orig	2014-02-10 19:30:18.044309275 +0100
+++ functions.php	2014-02-10 19:30:50.675111600 +0100
@@ -3007,11 +3007,26 @@
 	}
 }
 
+function login_redirect($returnto = '') {
+  $url = 'http://www.vassalengine.org/login.php';
+
+  if (strlen($returnto) > 0) {
+    $url .= '?returnto=' . urlencode($returnto);
+  }
+
+  header("Location: $url");
+}
+
 /**
 * Generate login box or verify password
 */
 function login_box($redirect = '', $l_explain = '', $l_success = '', $admin = false, $s_display = true)
 {
+  if (!$admin) {
+    login_redirect($_SERVER['REQUEST_URI']);
+    return;
+  }
+
 	global $db, $user, $template, $auth, $phpEx, $phpbb_root_path, $config;
 
 	if (!class_exists('phpbb_captcha_factory'))
