--- posting.php.orig	2014-02-10 19:28:12.805391025 +0100
+++ posting.php	2014-02-10 19:28:52.797600134 +0100
@@ -1140,6 +1140,10 @@
 			// The last parameter tells submit_post if search indexer has to be run
 			$redirect_url = submit_post($mode, $post_data['post_subject'], $post_author_name, $post_data['topic_type'], $poll, $data, $update_message, ($update_message || $update_subject) ? true : false);
 
+      /* start list bridge */
+      require_once('/var/www/bridge/src/forum_post_send.php');
+      /* end list bridge */
+
 			if ($config['enable_post_confirm'] && !$user->data['is_registered'] && (isset($captcha) && $captcha->is_solved() === true) && ($mode == 'post' || $mode == 'reply' || $mode == 'quote'))
 			{
 				$captcha->reset();
