--- mime.info.orig	2016-08-01 14:43:58.020380947 +0200
+++ mime.info	2016-08-01 14:46:16.519274431 +0200
@@ -77,6 +77,10 @@
 application/x-stuffit	[ARCHIVE]
 application/x-opc+zip	[ARCHIVE]
 
+application/vnd.vassalengine.game-log         [ARCHIVE]
+application/vnd.vassalengine.module           [ARCHIVE]
+application/vnd.vassalengine.module-extension [ARCHIVE]
+application/vnd.vassalengine.saved-game       [ARCHIVE]
 
 text/javascript application/x-javascript application/x-ecmascript text/ecmascript	[EXECUTABLE]
 application/x-bash	[EXECUTABLE]
