--- mime.info.orig	2014-09-09 22:54:02.642972586 +0200
+++ mime.info	2014-09-09 22:54:08.908330760 +0200
@@ -73,6 +73,10 @@
 application/x-stuffit	[ARCHIVE]
 application/x-opc+zip	[ARCHIVE]
 
+application/vnd.vassalengine.game-log         [ARCHIVE]
+application/vnd.vassalengine.module           [ARCHIVE]
+application/vnd.vassalengine.module-extension [ARCHIVE]
+application/vnd.vassalengine.saved-game       [ARCHIVE]
 
 text/javascript application/x-javascript application/x-ecmascript text/ecmascript	[EXECUTABLE]
 application/x-bash	[EXECUTABLE]
