--- mime.info.orig	2013-04-03 14:27:02.862668170 -0700
+++ mime.info	2013-04-03 14:29:33.665674316 -0700
@@ -69,6 +69,10 @@
 application/x-stuffit	[ARCHIVE]
 application/x-opc+zip	[ARCHIVE]
 
+application/vnd.vassalengine.game-log         [ARCHIVE]
+application/vnd.vassalengine.module           [ARCHIVE]
+application/vnd.vassalengine.module-extension [ARCHIVE]
+application/vnd.vassalengine.saved-game       [ARCHIVE]
 
 text/javascript application/x-javascript application/x-ecmascript text/ecmascript	[EXECUTABLE]
 application/x-bash	[EXECUTABLE]
