--- mime.info.orig	2015-06-19 20:23:28.288346790 +0200
+++ mime.info	2015-06-19 20:23:44.975279983 +0200
@@ -77,6 +77,10 @@
 application/x-stuffit	[ARCHIVE]
 application/x-opc+zip	[ARCHIVE]
 
+application/vnd.vassalengine.game-log         [ARCHIVE]
+application/vnd.vassalengine.module           [ARCHIVE]
+application/vnd.vassalengine.module-extension [ARCHIVE]
+application/vnd.vassalengine.saved-game       [ARCHIVE]
 
 text/javascript application/x-javascript application/x-ecmascript text/ecmascript	[EXECUTABLE]
 application/x-bash	[EXECUTABLE]
