--- mime.types.orig	2015-06-19 20:23:39.576978088 +0200
+++ mime.types	2015-06-19 20:23:52.215684893 +0200
@@ -16,6 +16,10 @@
 application/vnd.mif mif
 application/vnd.ms-excel xls xlt xla
 application/vnd.ms-powerpoint ppt pot pps ppa
+application/vnd.vassalengine.game-log vlog
+application/vnd.vassalengine.module vmod
+application/vnd.vassalengine.module-extension vmdx
+application/vnd.vassalengine.saved-game vsav
 application/vnd.wap.wbxml wbxml
 application/vnd.wap.wmlc wmlc
 application/vnd.wap.wmlscriptc wmlsc
@@ -61,7 +65,7 @@
 application/xslt+xml xslt
 application/xml xml xsl xsd kml
 application/xml-dtd dtd
-application/zip zip jar xpi sxc stc sxd std sxi sti sxm stm sxw stw
+application/zip zip jar xpi sxc stc sxd std sxi sti sxm stm sxw stw vmod vmdx vlog vsav
 application/x-rar rar
 audio/basic au snd
 audio/midi mid midi kar
@@ -114,7 +118,7 @@
 text/css css
 text/csv csv
 text/html html htm
-text/plain txt
+text/plain txt vdck
 text/richtext rtx
 text/rtf rtf
 text/sgml sgml sgm
