--- mime.types.orig	2014-09-09 22:53:35.732434349 +0200
+++ mime.types	2014-09-09 22:56:04.192923148 +0200
@@ -16,6 +16,10 @@
 application/vnd.mif mif
 application/vnd.ms-excel xls xlt xla
 application/vnd.ms-powerpoint ppt pot pps ppa
+application/vnd.vassalengine.game-log vlog
+application/vnd.vassalengine.module vmod
+application/vnd.vassalengine.module-extension vmdx
+application/vnd.vassalengine.saved-game vsav
 application/vnd.wap.wbxml wbxml
 application/vnd.wap.wmlc wmlc
 application/vnd.wap.wmlscriptc wmlsc
@@ -59,7 +63,7 @@
 application/xslt+xml xslt
 application/xml xml xsl xsd kml
 application/xml-dtd dtd
-application/zip zip jar xpi sxc stc sxd std sxi sti sxm stm sxw stw
+application/zip zip jar xpi sxc stc sxd std sxi sti sxm stm sxw stw vmod vmdx vlog vsav
 application/x-rar rar
 audio/basic au snd
 audio/midi mid midi kar
@@ -110,7 +114,7 @@
 text/calendar ics ifb
 text/css css
 text/html html htm
-text/plain txt
+text/plain txt vdck
 text/richtext rtx
 text/rtf rtf
 text/sgml sgml sgm
