--- LdapAuthentication.php	2015-06-16 23:10:35.000000000 +0200
+++ LdapAuthentication.php	2015-08-23 22:53:03.386673783 +0200
@@ -597,7 +597,7 @@
 			$servers = $servers . " " . $serverpre . $tok . ":" . $this->getConf( 'Port', $domain );
 			$tok = strtok( " " );
 		}
-		$servers = rtrim( $servers );
+		$servers = trim( $servers );
 
 		$this->printDebug( "Using servers: $servers", SENSITIVE );
 
